docs/features/blocks/contentblocks/googlemap.rst

```diff
@@ -0,0 +1,33 @@
+Google Map Block
+================
+
+Allows the user to add a Google Map location and display the location on the map. 
+
+Field Reference
+---------------
+
+Fields and purposes:
+
+* **Search Query** - Address or search term used to find your location on the map.
+
+* **Map Title** - Map title for screen readers, ex: "Map to Simple Sweet Desserts"
+
+* **Google Place ID** - Place IDs uniquely identify a place in the Google Places database and on Google Maps
+
+.. note::
+    Google Place IDs require a Google API Key. Read more about `Google Place IDs <https://developers.google.com/places/place-id>`_.
```

Comment:
Note can be moved down below the fields, and say that both place ID and zoom level require API key.

Reply:
done